Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerCommandCollectionViewCell_Tests.swift

```diff
@@ -2,25 +2,94 @@
 // Copyright © 2021 Stream.io Inc. All rights reserved.
 //
 
+import SnapshotTesting
+@testable import StreamChat
 @testable import StreamChatUI
 import XCTest
 
-class ChatMessageComposerCommandCellView_Tests: XCTestCase {
+class ChatMessageComposerCommandCollectionViewCell_Tests: XCTestCase {
+    /// Default width and height for the cell.
+    private let defaultCellSize: CGSize = CGSize(width: 250, height: 50)
+    
+    /// Default content of the view
+    private let defaultCommand = Command(name: "Default", description: "", set: "", args: "[text]")
+
     func test_defaultAppearance() {
-        let view = ChatMessageComposerCommandCellView()
-        view.command = .init(name: "Hello", description: "what", set: "is this", args: "WHAT")
-        AssertSnapshot(view, size: CGSize(width: 300, height: 40), record: true)
+        let cell = ChatMessageComposerCommandCollectionViewCell()
+        cell.commandView.content = defaultCommand
+        AssertSnapshot(cell, size: defaultCellSize)
     }
 
-    func test_defaultAppearance_withDirectMessageChannel() {}
+    func test_defaultAppearance_allAvailableCommands() {
+        let cell = ChatMessageComposerCommandCollectionViewCell()
+        cell.uiConfig.images.commandIcons.forEach { command in
+            cell.commandView.content = .init(name: command.key, description: "", set: "", args: "[text]")
+            AssertSnapshot(cell, variants: [.defaultLight], size: defaultCellSize, suffix: command.key)
+        }
```

Comment:
What exactly is this test testing?

Reply:
This is testing all the commands we support. Imagine, if we edit the commands in the future, and for a "Key" the image is corrupted or something, this test will catch that. This is making sure that all commands have a corresponding image

Reply:
IMHO those snapshot tests should catch only layout stuff and how we are doing with handling dynamic fonts and accessibility. I get what you are trying to solve with the corresponding image corrupted/different, but I think it's unnecessary and crosses the scope of Snapshot tests with something else.

I mean you have really good point, but this should be tested somewhere else / somehow else.  But this is interesting topic actually 🤔

Reply:
I think you can use snapshot tests to test whatever you want. That is why we should also test state differences. You can even test other things, not only images. I don't think we should limit ourselves to test only layout and accessibility issues with snapshots. We can test also business logic. And in this case actually, by testing all commands we support, we are checking that each command does not change the layout in someway 🤔

Reply:
Resolved, removed the test ✅

Reply:
New test added ✅  @VojtaStavik